generate.R

```diff
@@ -57,7 +56,7 @@ generate_video <- function(srt_file, duration = "00:00", overwrite = "auto") {
     text_to_frame(df$text[i], file.path(captions_dir, paste0(df$n[i], ".png")))
   }
 
-  fps <- 24
+  fps <- 1
```

Comment:
I made this 1 to speed up testing; I'm not sure if it makes an appreciable difference to the output or not.